docs/scaling.md

```diff
@@ -12,12 +12,37 @@ As of v3.0, the controller supports having a hot-standby for [High Availability]
 
 ## Vertically Scaling
 
-You can scale the controller vertically:
+You can scale the controller vertically in these ways:
 
-- If you have many workflows, increase `--workflow-workers` and `--workflow-ttl-workers`.
-- Increase both `--qps` and `--burst`.
+### Container Resource Requests
 
-You will need to increase the controller's memory and CPU.
+If you observe the Controller using its total request CPU or memory, you should increase those.
```

Comment:
```suggestion
If you observe the Controller using its total CPU or memory requests, you should increase those.
```

"resource requests" is the [usual](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-requests-and-limits-of-pod-and-container) order, so move "requests" to after CPU/mem